etc/kayobe/kolla/globals.yml

```diff
@@ -9,62 +9,15 @@ enable_docker_repo: "{% raw %}{{ 'overcloud' not in group_names or ansible_facts
+# with signature: `Missing section footer for 0000:00:01.3/piix4_pm`. Test nova carefully before bumping.
 kayobe_image_tags:
   openstack:
-    centos: yoga-20230905T130221
-    rocky: yoga-20230310T170929
-    ubuntu: yoga-20230913T160015
-  bifrost:
-    rocky: yoga-20230310T194732
-  blazar:
-    rocky: yoga-20230315T130918
-  caso:
-    rocky: yoga-20230315T130918
-  grafana:
-    rocky: yoga-20230419T111514
-  ironic:
-    rocky: yoga-20230316T170311
-  ironic_dnsmasq:
-    rocky: yoga-20230310T170929
-  kibana:
-    centos: yoga-20230907T152300
-  neutron:
-    rocky: yoga-20230728T081242
-  nova:
-    rocky: yoga-20230718T112646
-  nova_libvirt:
-    # Live-migration is broken from qemu-kvm-6.2.0-20.module_el8.7.0+1218+f626c2ff to qemu-kvm-6.2.0-39.module_el8+669+76cc32af
-    # with signature: `Missing section footer for 0000:00:01.3/piix4_pm`. Test carefully before bumping.
-    centos: yoga-20230718T112646
-  openvswitch:
-    centos: yoga-20231019T102525
-    rocky: yoga-20231019T102525
-  ovn:
-    centos: yoga-20231019T102525
-    rocky: yoga-20231019T102525
-  prometheus_node_exporter:
-    rocky: yoga-20230315T170614
-
-bifrost_tag: "{% raw %}{{ kayobe_image_tags['bifrost'][kolla_base_distro] | default(openstack_tag) }}{% endraw %}"
-blazar_tag: "{% raw %}{{ kayobe_image_tags['blazar'][kolla_base_distro] | default(openstack_tag) }}{% endraw %}"
-caso_tag: "{% raw %}{{ kayobe_image_tags['caso'][kolla_base_distro] | default(openstack_tag) }}{% endraw %}"
-grafana_tag: "{% raw %}{{ kayobe_image_tags['grafana'][kolla_base_distro] | default(openstack_tag) }}{% endraw %}"
-ironic_tag: "{% raw %}{{ kayobe_image_tags['ironic'][kolla_base_distro] | default(openstack_tag) }}{% endraw %}"
-ironic_dnsmasq_tag: "{% raw %}{{ kayobe_image_tags['ironic_dnsmasq'][kolla_base_distro] | default(openstack_tag) }}{% endraw %}"
-kibana_tag: "{% raw %}{{ kayobe_image_tags['kibana'][kolla_base_distro] | default(openstack_tag) }}{% endraw %}"
-neutron_tag: "{% raw %}{{ kayobe_image_tags['neutron'][kolla_base_distro] | default(openstack_tag) }}{% endraw %}"
-nova_tag: "{% raw %}{{ kayobe_image_tags['nova'][kolla_base_distro] | default(openstack_tag) }}{% endraw %}"
-nova_libvirt_tag: "{% raw %}{{ kayobe_image_tags['nova_libvirt'][kolla_base_distro] | default(nova_tag) }}{% endraw %}"
-openvswitch_tag: "{% raw %}{{ kayobe_image_tags['openvswitch'][kolla_base_distro] | default(openstack_tag) }}{% endraw %}"
-ovn_tag: "{% raw %}{{ kayobe_image_tags['ovn'][kolla_base_distro] | default(openstack_tag) }}{% endraw %}"
-prometheus_node_exporter_tag: "{% raw %}{{ kayobe_image_tags['prometheus_node_exporter'][kolla_base_distro] | default(openstack_tag) }}{% endraw %}"
-
-glance_tls_proxy_tag: "{% raw %}{{ haproxy_tag | default(openstack_tag) }}{% endraw %}"
-neutron_tls_proxy_tag: "{% raw %}{{ haproxy_tag | default(openstack_tag) }}{% endraw %}"
```

Comment:
It's best to keep these because it's not obvious that you need them if setting haproxy_tag. Could include a comment to that effect.

Reply:
Will do. I've set up a PR for fixes [here](https://github.com/stackhpc/stackhpc-kayobe-config/pull/723) where I'll make that change